src/sentry/auth/superuser.py

```diff
@@ -70,6 +75,15 @@ class Superuser:
 
     org_id = ORG_ID
 
+    def _check_expired_on_org_change(self):
+        if hasattr(self, "expires") and self.expires is not None:
```

Comment:
why do you need to check if this attribute exists with `hasattr`?

Reply:
i  added this since expires is not initialized at the beginning and only set when _set_logged_in is called. I added this since certain test cases before tested the expiry time without actually calling set_logged_in, in which this would error out since self.expires doesn't exist. As far as i know, i don't think it would happen on prod but as it happens in the tests i was paranoid